internal/go/skycfg/proto_message.go

```diff
@@ -23,7 +23,9 @@ import (
 	"reflect"
 	"sort"
 	"strings"
+	"time"
 
+	"github.com/gogo/protobuf/types"
```

Comment:
Lets use the standard go-protobuf here so we don't need to drag in a hard dependency on GoGo: https://godoc.org/github.com/golang/protobuf/ptypes#DurationProto

Reply:
Hm, this would mean user must make sure to return `google.protobuf.Duration`-reflected type from the standard registry, not gogo (which they are probably going to be using since it's a gogo-only option).

Reply:
If I have skycfg code like this:

```python
envoy = proto.package("envoy.api.v2")
pb = proto.package("google.protobuf")

cluster = v2.Cluster(connect_timeout = pb.Duration(seconds = 1))
```

Then the `pb.Duration` is probably being resolved to the go-protobuf implementation. It will be confusing if we need to add a new lookup function or option to make `pb.Duration()` specifically return the GoGo version.

Similarly, `cluster.connect_timeout` ought to return the normal types instead of the GoGo types so that users don't have to contend with:

```python
ct = cluster.connect_timeout
cluster.connect_timeout = ct # TypeError, wtf?
```

The part we can't solve, and this is part of why the existence of GoGo is so frustrating, is that GoGo-generated code can have different duration types even in the same message. I don't think there's anything in Envoy or Kubernetes that does such a thing, so we can worry about it later.

Reply:
> Then the pb.Duration is probably being resolved to the go-protobuf implementation.

That actually will depends on which registry does the lookup first =) so one would need to make sure they implemented their custom registry as follows (or just use the default registry):

```
func (*protoRegistry) UnstableProtoMessageType(name string) (reflect.Type, error) {
	if t := proto.MessageType(name); t != nil {
		return t, nil
	}
	if t := gogo_proto.MessageType(name); t != nil {
		return t, nil
	}
	return nil, nil
}
```

Reply:
Done.